### DISCUSSION TITLE
Synthetic Aperture Radar (SAR) Simulator (Project 211)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#43] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 211](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Synthetic%20Aperture%20Radar%20(SAR)%20Simulator)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by VimsRocz** [comment]
Hello Robert

Hi, I am a student at Uni Bremen and interested in working and collaborating on this project. We have already started. Thanks
